packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -62,6 +74,26 @@ function ProductDetails({ context: staleProduct }: Props) {
     },
   } = data
 
+  const [skuPropertyName] = useMemo(() => {
+    return Object.keys(skuVariants.activeVariations)
+  }, [skuVariants.activeVariations])
+
+  const mountItemHref = useCallback(
+    (option: SkuOption) => {
+      const currentOptionName = skuPropertyName ?? option.label.split(':')[0]
+      const currentItemHref = `/${getSkuSlug(
+        skuVariants.slugsMap,
+        {
+          ...skuVariants.activeVariations,
+          [currentOptionName]: option.value,
+        },
+        skuPropertyName
+      )}/p`
+      return currentItemHref
+    },
+    [skuPropertyName, skuVariants.activeVariations, skuVariants.slugsMap]
+  )
```

Comment:
Same renaming comment (just commenting on all I notice to help spot them hahahah)